public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -1,56 +1,39 @@
-import { css } from '@emotion/css';
+import { EmptyState, LinkButton, Stack, TextLink } from '@grafana/ui';
 
-import { GrafanaTheme2 } from '@grafana/data';
-import { CallToActionCard, useStyles2, Stack } from '@grafana/ui';
-import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
-
-import { logInfo, LogMessages } from '../../Analytics';
 import { useRulesAccess } from '../../utils/accessControlHooks';
 
 export const NoRulesSplash = () => {
   const { canCreateGrafanaRules, canCreateCloudRules } = useRulesAccess();
-  const styles = useStyles2(getStyles);
-  if (canCreateGrafanaRules || canCreateCloudRules) {
-    return (
-      <div>
-        <p>{"You haven't created any alert rules yet"}</p>
-        <Stack gap={1}>
-          <div className={styles.newRuleCard}>
-            <EmptyListCTA
-              title=""
-              buttonIcon="bell"
-              buttonLink={'alerting/new/alerting'}
-              buttonTitle="New alert rule"
-              proTip="you can also create alert rules from existing panels and queries."
-              proTipLink="https://grafana.com/docs/"
-              proTipLinkTitle="Learn more"
-              proTipTarget="_blank"
-              onClick={() => logInfo(LogMessages.alertRuleFromScratch)}
-            />
-          </div>
+  const canCreateAnything = canCreateGrafanaRules && canCreateCloudRules;
```

Comment:
what happens when you can create only one type? for example, if user can only create Grafana managed rules?

Reply:
Good question; I think it makes sense to distinguish these two – I'll amend that to the PR 👍